.github/CODEOWNERS

```diff
@@ -1,16 +1,14 @@
+# NOTE TO DEVELOPERS: Procedure for picking code owners:
```

Comment:
Please put this as a new paragraph below the existing fileoverview (line 10).

Let's also rephrase this a bit:

```
IMPORTANT NOTE TO DEVELOPERS: 
- When assigning codeowners for new files, please do so by picking an appropriate team lead from this page: ... 
- You do not need to assign yourself as a codeowner (unless you are a project lead listed on the page above).
```

Reply:
done!